eox_tagging/api/v1/viewset.py

```diff
@@ -123,10 +123,16 @@
             str,
             "Shortcut to filter objects of target_type `user` with id `username`.",
         ),
+        query_parameter(
+            "owner_type",
+            str,
+            "Shortcut to filter objects of owner_type `user` or `site`.",
+        ),
         query_parameter(
             "target_type",
             str,
-            "The type of the object that was tagged, one of: `course`, `courseenrollment`, `user`",
+            "The type of the object that was tagged, one of: `course (opaquekeyproxymodel)`"
```

Comment:
How many people actually know what `opaquekeyproxymodel` means?

Reply:
Not many. My issue here is: if you filter using just `course` it's not going to work.

Reply:
Let me rephrase the change